src/router/index.js

```diff
@@ -3,29 +3,35 @@ import VueRouter from "vue-router";
 import OverviewPage from "../views/village/OverviewPage";
 import MapPage from "../views/map/MapPage";
 import LoginPage from "../views/login/LoginPage";
+import EmailVerification from "../views/emailVerification/EmailVerification"
 
 Vue.use(VueRouter);
 
 const routes = [
-  {
-    path: "/",
-    name: "OverviewPage",
-    component: OverviewPage,
-  },
-  {
-    path: "/map",
-    name: "MapPage",
-    component: MapPage,
-  },
-  {
-    path: "/login",
-    name: "LoginPage",
-    component: LoginPage,
-  },
+    {
+        path: "/",
+        name: "OverviewPage",
+        component: OverviewPage,
+    },
+    {
+        path: "/map",
+        name: "MapPage",
+        component: MapPage,
+    },
+    {
+        path: "/login",
+        name: "LoginPage",
+        component: LoginPage,
+    },
+    {
+        path: "/emailVerification",
```

Comment:
one think, how you would to pass verification code here? I mean, you can copy from i.e from email, and paste in this view, but for UX purposes you could pass code via query param. To handle it on vue router, you should change path there to `/emailVerification/:code`. After that, you will be able to get this code directly from your code

Reply:
`this.$route.params.code` it will be getter for code